Core/Service/Configuration.php

```diff
@@ -111,9 +139,14 @@ protected function loadConfiguration($configurationId)
     public function setOrdered($configurationId, $app)
     {
         $response = oxNew(Request::class)->setOrdered($configurationId, $app);
-        if ( $response ) {
+        if ($response) {
             Registry::get(Logger::class)->info('set ordered result', [$response]);
             return json_decode($response);
         }
     }
-}
\ No newline at end of file
+
+    private function calculateCombinedPrice($basketArticles): int
+    {
+        return $basketArticles[0]["price"];
```

Comment:
Ist das richtig? Es können ja mehrre Artikel in der Stückliste der Konfiguration sein.
Normalerweise nehmen wir den Preis der Konfigratrion - ist das das hier?

Reply:
Dies ist korrekt, hier kommt der Gesamtpreis der Konfiguration an.

Reply:
Entschuldige bitte...
Das glaub ich nicht.
Schau mal hier im Shopware Plugin:
https://github.com/OC-Git/k3-shopware6/blob/main/src/Storefront/Controller/APIController.php#L263

Hast du probiert, zwei Artikel in die Stückliste zu legen?
Dann soll der Preis im Warenkorb ja die Summe der Artikelpreise sein.
Bei dir würde es ja nur der Preis des ersten sein.

Glaube wirklich, du musst das Feld "price" von der Konfiguration benutzen.

Reply:
Entschuldige bitte...
Das glaub ich nicht.
Schau mal hier im Shopware Plugin:
https://github.com/OC-Git/k3-shopware6/blob/main/src/Storefront/Controller/APIController.php#L263

Hast du probiert, zwei Artikel in die Stückliste zu legen?
Dann soll der Preis im Warenkorb ja die Summe der Artikelpreise sein.
Bei dir würde es ja nur der Preis des ersten sein.

Glaube wirklich, du musst das Feld "price" von der Konfiguration benutzen.